src/pages/academy/Academy.tsx

```diff
@@ -67,6 +68,7 @@ const Academy: React.FC<{}> = () => {
           />
         ))}
         {enableGame && <Route path={'game'} element={<Game />} />}
+        <Route path={`/notipreference`} element={<NotiPreference />} />
```

Comment:
Do we need the leading slash here? Also, this can be a plain string instead of a template literal:

```suggestion
        <Route path="notipreference" element={<NotiPreference />} />
```

Reply:
Nice catch! I think it was changed after the react-router upgrade but we fixed it for the other routes as well.